### DISCUSSION TITLE
Intuition for `+++`

### DISCUSSION THREAD

**Discussion by tusharmath** [dream11/zio-http#251] [Ideas]
Combining two http apps can be done as follows — 

```scala
val h1: Http[_, _, _, _] = ???
val h2: Http[_, _, _, _] = ???
val h3: Http[_, _, _, _] = ???

val h123 = h1 defaultWith h2 defaultWith h3

// Using operators
val h123 = h1 +++ h2 +++ h3
```

The way `defaultWith` works is that when the response of `h1` is `Empty` thats when `h2` is executed. Similarly if `h2` resolves to an `Empty`, only then `h3` is going to be executed. 

The named operator `defaultWith` makes a lot of sense for this kind of operation, but the symbol equivalent `+++` doesn't put across a similar intuition. The **proposed** operator for this use case IMO should be `+>`. Thoughts anyone?

**Comment by ghost** [comment]
The operator defaultWith should do the job, 

+>  would be a good choice, but there is a chance that without proper documentation the proposed operator can be mistaken with zipRight Operator from ZIO.  

try considering ~> operator which will be more meaningful in my opinion.
